src/draw/lv_draw_buf.h

```diff
@@ -91,6 +85,14 @@ typedef void (*lv_draw_buf_cache_operation_cb)(const lv_draw_buf_t * draw_buf, c
 
 typedef uint32_t (*lv_draw_buf_width_to_stride_cb)(uint32_t w, lv_color_format_t color_format);
 
+struct _lv_draw_buf_t {
```

Comment:
We are trying to avoid `_lv_*` now

Reply:
You are right. Should I change it to `struct lv_draw_buf_t`?

Reply:
Looks good now, thanks!